src/Routing/AuthenticatedContentRouteSubscriber.php

```diff
@@ -53,6 +53,23 @@ protected function alterRoutes(RouteCollection $collection) {
         $route->setRequirement('_custom_access', 'tide_authenticated_content.access_jsonapi_checker::access');
       }
     }
+    $route_config = \Drupal::config('tide_authenticated_content.route_settings');
```

Comment:
Please use dependency injection. `\Drupal::` helper class is only used in static methods or where DI is not supported.